testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesExecutorTest.java

```diff
@@ -1541,4 +1569,164 @@ public void testSecureParContentsExecutorWithRequestObject() throws Exception {
         successfulLoginAndLogout(TEST_CLIENT, TEST_CLIENT_SECRET);
 
     }
+
+    @Test
+    public void testDPoPBindEnforcerExecutor() throws Exception {
```

Comment:
Minor comment, but is it possible to add this test into `DPoPTest` ? As `ClientPoliciesTest` is already quite big and can be nice to have all DPoP related things in the separate place as "DPoP" is still preview and not a core functionality (at least yet).

Reply:
Yes, I moved a test method to `DPoPTest`.